CHANGELOG.md

```diff
@@ -5,20 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.7.0 - 2019-02-09
```

Comment:
We need to mark that we removed `lmss`, `dnditemtable`, and the package options.

`hang` is a dependency again, as well.

Reply:
Done.